docs/user-guide/rs.rst

```diff
@@ -0,0 +1,230 @@
+   && python -u examples/nlp/gpt/serve_reward_model.py \
+      --config-path=${CRITIC_CONFIG_PATH} \
+      --config-name=${CRITIC_CONFIG_NAME} \
+      trainer.num_nodes=1 \
+      trainer.devices=8 \
+      ++model.tensor_model_parallel_size=4 \
+      rm_model_file=${RM_NEMO_FILE} \
+      inference.port=${CRITIC_PORT}
+   EOF
+
+   srun --het-group=0 -o $CRITIC_OUTFILE -e $CRITIC_ERRFILE --container-image=${CONTAINER} $MOUNTS bash -c "${cmd_critic_inference}" &
+
+   sleep 30
+
+   ACTOR_LOG_DIR="${RESULTS_DIR}/actor_results"
+   CHECKPOINT_DIR="${ACTOR_LOG_DIR}/checkpoints"
+   TENSOBOARD_DIR="${ACTOR_LOG_DIR}/tensorboard"
+
+   NUM_ROLLOUTS=16
+   NORMALIZE="True"
+   ACTOR_LR="1e-6"
+   ACTOR_GBS=16
+
+   mkdir -p $ACTOR_LOG_DIR
+   mkdir -p $TENSOBOARD_DIR
+   mkdir -p $CHECKPOINT_DIR
+
+   ACTOR_NAME="${NAME}_actor"
+
+   host_critic="$(scontrol show hostnames=$SLURM_JOB_NODELIST_HET_GROUP_0 | head -n1)"
+
+   read -r -d '' cmd_rs <<EOF
+   cd ${GPFS} \
+   && export PYTHONPATH="${GPFS}:${PYTHONPATH}" \
+   && export HYDRA_FULL_ERROR=1 \
+   && python -u examples/nlp/gpt/train_gpt_rs_actor.py \
+      --config-path=${CONF_DIR} \
+      --config-name=${CONFIG_NAME} \
+      "model.data.data_prefix={train: [${TRAIN_DATA_PATH}], validation: [${VALID_DATA_PATH}], test: [${VALID_DATA_PATH}]}" \
+      pretrained_checkpoint.restore_from_path=\"${ACTOR_NEMO_FILE}\" \
+      exp_manager.checkpoint_callback_params.save_top_k=1 \
+      exp_manager.explicit_log_dir=\"${ACTOR_LOG_DIR}\" \
+      trainer.rs.max_epochs=1 \
+      trainer.rs.max_steps=313 \
+      trainer.rs.val_check_interval=4 \
+      trainer.num_nodes=1 \
+      trainer.devices=8 \
+      ++model.tensor_model_parallel_size=4 \
+      model.global_batch_size=${ACTOR_GBS} \
+      model.micro_batch_size=1 \
+      model.optim.lr=\"\\\$\{multiply:${ACTOR_LR},1.001\}\" \
+      model.optim.sched.warmup_steps=0 \
+      model.optim.sched.constant_steps=312 \
+      model.optim.sched.min_lr=${ACTOR_LR} \
+      model.optim.weight_decay=0.01 \
+      model.rs.num_rollout_samples=${NUM_ROLLOUTS} \
+      model.rs.rollout_micro_batch_size=2 \
+      model.rs.forward_micro_batch_size=2 \
+      model.rs.val_rollout_micro_batch_size=2 \
+      model.data.data_impl=jsonl \
+      remote_rm.reward_model.ip=${host_critic} \
+      remote_rm.reward_model.port=${CRITIC_PORT} \
+      model.rs.num_rollouts_per_prompt=8 \
+      model.rs.top_n_rollouts=1
+   EOF
+
+   srun --het-group=1 -o $PPO_OUTFILE -e $PPO_ERRFILE --container-image=${CONTAINER} $MOUNTS bash -c "${cmd_rs}" &
+
+   wait
+
+The above script runs the reward model server on 1 node and the actor on 1 node.
+
+It is important to launch all jobs with ``&`` after the srun command, to ensure they do not block each other. 
+
+.. note::
+   Make sure to change the reward model arg ``trainer.rs.inference_micro_batch_size`` such that ``trainer.rs.inference_micro_batch_size * DP size <= model.rs.rollout_micro_batch_size``.
+
+RS Results
```

Comment:
revise the title. Ensure all words are capitalized. We want to spell out the title for the purposes to support online searches and SEO for our content.

Rejection Sampling Results

Reply:
Revised the title.